OpenSim/Simulation/Model/Muscle.cpp

```diff
@@ -144,6 +157,7 @@ void Muscle::constructProperties()
     // By default the min and max controls on muscle are 0.0 and 1.0
     setMinControl(0.0);
     setMaxControl(1.0);
+    upd_GeometryPath().setDefaultColor(DefaultMuscleColor);
```

Comment:
The argument was made that muscles should show red by default, but not other forces that use GeometryPath, that's why the color setting can't be made by the GeometryPath itself.